torch/csrc/jit/script/compiler.cpp

```diff
@@ -584,32 +590,34 @@ struct to_ir {
     }
     return emitNode(
                Symbol("type_as"),
-               {emitExpr(input, 1)[0], createConstant(at::CPU(t).ones({1}))},
+               input,
+               {emitExpr(input, 1)[0], createConstant(input, at::CPU(t).ones({1}))},
                1)
         ->outputs();
   }
 
-  std::vector<Value*> emitConst(const double val, const std::string& type) {
+  std::vector<Value*> emitConst(const TreeRef& loc, const double val, const std::string& type) {
```

Comment:
It would be nicer to avoid passing TreeRefs around, and only give the `SourceLocation`

Reply:
Yep, fixed it.